package.json

```diff
@@ -137,11 +137,6 @@
 		"playwright": "^1.29.2",
 		"xo": "^0.53.1"
 	},
-	"resolutions": {
-		"glob-parent": "5.1.2",
-		"minimist": "1.2.7",
-		"yargs-parser": "21.1.1"
-	},
```

Comment:
Not sure if all resolutions were outdated, but it should be ok to clean them, we'll see if some new resolutions are needed.